service/gcs/core/gcs/gcs.go

```diff
@@ -314,7 +316,7 @@ func (c *gcsCore) ExecProcess(id string, params prot.ProcessParameters, connecti
 
 	containerEntry := c.getContainer(id)
 	if containerEntry == nil {
-		return -1, errors.WithStack(gcserr.NewContainerDoesNotExistError(id))
+		return -1, nil, errors.WithStack(gcserr.NewContainerDoesNotExistError(id))
```

Comment:
This is stupid. We dont need a stack for this error.

Reply:
Fixed (but not my code :innocent:)